MSAL/src/native_auth/controllers/sign_up/MSALNativeAuthSignUpController.swift

```diff
@@ -204,9 +204,9 @@ final class MSALNativeAuthSignUpController: MSALNativeAuthBaseController, MSALNa
                 errorUri: apiError.errorURI
             )
             stopTelemetryEvent(event, context: context, error: error)
-            MSALLogger.log(level: .error,
-                           context: context,
-                           format: "InvalidUsername in signup/start request \(error.errorDescription ?? "No error description")")
+            MSALLogger.logPII(level: .error,
+                              context: context,
+                              format: "InvalidUsername in signup/start request \(MSALLogMask.maskPII(error.errorDescription))")
```

Comment:
Username is EUII, can you use the correct mark? Thanks

Reply:
Done, thank you.